defaults/main.yml

```diff
@@ -22,13 +23,16 @@ php_post_max_size: 8M
 
 php_ini_configuration_template:
   src: php.ini.j2
-  dest: "{{ php_install_dir }}/lib/php.ini"
+  dest: "{{ php_include_ini_dir }}/php.ini"
```

Comment:
I think php.ini must be retained in the previous dir, it's an standard and we can leave php_include_ini_dir only for additional files.